modules/ROOT/partials/configuration/ai_shortcuts.adoc

```diff
@@ -99,14 +99,31 @@ tinymce.init({
 
 === Valid Shortcuts
 
-Valid shortcut objects contain the following fields.
+Valid shortcut objects contain the following properties.
 
 `+title+`:: A string which is displayed in the `+aishortcuts+` toolbar button and menu item. This will indicate which shortcut is used, or which category of shortcuts are in this menu.
 
 And either
 
-`+prompt+`:: A string containing the query which is given to the `+ai_request+` function when the shortcut is used.
+`+subprompts+`:: An array containing more valid shortcut objects. This allows shortcuts to be grouped into categories within the {pluginname} shortcuts toolbar button and menu item.
 
 or
 
-`+subprompts+`:: An array containing more valid shortcut objects. This allows shortcuts to be grouped into categories within the {pluginname} shortcuts toolbar button and menu item.
\ No newline at end of file
+`+prompt+`:: A string containing the query which is given to the `+ai_request+` function when the shortcut is used.
+
+
+==== The `+selection+` property
```

Comment:
Do we need to specify what the default value is?

Reply:
It can be undefined